src/main/kotlin/net/ccbluex/liquidbounce/event/EventListener.kt

```diff
@@ -66,6 +66,21 @@ interface EventListener {
         }
     }
 
+    /**
+     * Cancels all sequences associated with this event listener.
+     * This is called when a module is disabled to ensure no sequences continue running.
+     */
+    fun cancelAllSequences() {
+        SequenceManager.sequences.removeAll { sequence ->
+            if (sequence.owner == this) {
+                sequence.cancel()
+                true
+            } else {
+                false
+            }
+        }
+    }
```

Comment:
This is a feature of the sequence manager, but it is implemented in EventListener.

It should be implemented in Manager

Reply:
I addressed this in https://github.com/CCBlueX/LiquidBounce/pull/5054/commits/69d5b0feb55e0e492557ea16b0c030795670a1c2